### DISCUSSION TITLE
Target #48🧼

### DISCUSSION THREAD

**Discussion by pyotrpopov443** [cssbattle/forum#216] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45715791/149625931-328f4642-d89a-4c97-90f0-97891cc5a5af.png)
I can't figure out why my inner shadow doesn't have rounded corners. If that were the case, I would have 100%. Do you have any solution for this?

**Comment by SelenIT** [comment, marked as answer]
Corner radii of the shadows are increased (for external shadows) or decreased (for internal shadows) by the spread distance values. You can see it by varying the spread distance of the inset shadow between `0` and `32q`. When spread distance becomes equal or greater than `border-radius` (in this case, `21q`), corner radii of the inset shadow shrink to 0.

The only workaround I can figure out for the single-element solution is using 4 inset shadows with no spread instead, shifted in opposite directions. Of course, it would make a solution much longer. So maybe it would be worth employing another element for the central bar?..

**Reply by pyotrpopov443** [reply]
Ok, thank you! I will try to get max score with extra element.

**Reply by SelenIT** [reply]
You don't have to _add_ it, actually. Only to reorganize the code to get access to _both_ elements you already have :)

**Reply by pyotrpopov443** [reply]
Thanks! I got it, 142 chars already and 100%.
